src/audio/google/google_ctc_audio_processing.toml

```diff
@@ -0,0 +1,17 @@
+	// # CTC module config
+	[[module.entry]]
+	name = "CTC"
+	uuid = "BF0E1BBC-DC6A-45FE-BC90-2554CB137AB4"
+	affinity_mask = "0x1"
+	instance_count = "40"
+	domain_types = "0"
+	load_type = "0"
+	module_type = "9"
+	auto_start = "0"
+	sched_caps = [1, 0x00008000]
+	// # pin = [dir, type, sample rate, size, container, channel-cfg]
+	pin = [0, 0, 0xfeef, 0xf, 0xf, 0x45ff, 1, 0, 0xfeef, 0xf, 0xf, 0x1ff]
+	// # mod_cfg [PAR_0 PAR_1 PAR_2 PAR_3 IS_BYTES CPS IBS OBS MOD_FLAGS CPC OBLS]
```

Comment:
This drops comments from the pre-processed file. You can use the new `REM #` prefix instead to avoid that.